src/Rules/Api/NodeConnectingVisitorAttributesRule.php

```diff
@@ -42,10 +43,11 @@ public function processNode(Node $node, Scope $scope): array
 			return [];
 		}
 		$argType = $scope->getType($args[0]->value);
-		if (!$argType instanceof ConstantStringType) {
+		if (count($argType->getConstantStrings()) === 0) {
```

Comment:
maybe better

```suggestion
		if (count($argType->getConstantStrings()) !== 1) {
```

instead of ignoring possible known values

Reply:
Thanks!